lib/engine_http.js

```diff
@@ -130,8 +130,12 @@ HttpEngine.prototype.step = function step(requestSpec, ee) {
       requestParams.agent = context._agent;
     }
 
-    if (params.followRedirect == false) {
-      requestParams.followRedirect = false
+    if (params.noRedirect != undefined) {
```

Comment:
The global option should be set in `config.http.defaults` and over-ridden by a property on the `requestSpec` if set.

Scenario-level option could be omitted for now.